easybuild/easyblocks/g/gromacs.py

```diff
@@ -59,7 +59,7 @@ class EB_GROMACS(CMakeMake):
     @staticmethod
     def extra_options():
         extra_vars = {
-            'double_precision': [False, "Build with double precision enabled (-DGMX_DOUBLE=ON)", CUSTOM],
+            'double_precision': [None, "Build with double precision enabled (-DGMX_DOUBLE=ON)", CUSTOM],
```

Comment:
A note on what `None` means would be good. I couldn't tell if what is done below is correct as I don't know what `None` is supposed to do